<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>autoclicker</class>
 <widget class="QMainWindow" name="autoclicker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>267</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>348</width>
    <height>267</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>348</width>
    <height>267</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Auto Clicker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="clickIntervalGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Tıklama Aralığı</string>
    </property>
    <widget class="QSpinBox" name="hourSpinBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>81</width>
       <height>29</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="minuteSpinBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>81</width>
       <height>29</height>
      </rect>
     </property>
     <property name="maximum">
      <number>59</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="secondSpinBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>81</width>
       <height>29</height>
      </rect>
     </property>
     <property name="maximum">
      <number>59</number>
     </property>
    </widget>
    <widget class="QLabel" name="minuteLabel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Dakika</string>
     </property>
    </widget>
    <widget class="QLabel" name="secondLabel">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Saniye</string>
     </property>
    </widget>
    <widget class="QLabel" name="hourLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Saat</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="mauseActionGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Mouse Aksiyonu</string>
    </property>
    <widget class="QComboBox" name="buttonComboBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Sol Buton</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sağ Buton</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="actionComboBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Tek Tık</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Çift Tık</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="mauseButtonLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>33</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Mouse</string>
     </property>
    </widget>
    <widget class="QLabel" name="mauseActionLabel">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>73</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Aksiyon</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Başlat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Durdur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Çıkış</string>
    </property>
   </widget>
   <widget class="QComboBox" name="LanguageComboBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>82</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>2</number>
    </property>
    <property name="pageStep">
     <number>0</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
    <property name="sliderPosition">
     <number>2</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
